### PR TITLE
Rename redirect and flash rebase

### PR DESCRIPTION
**Pull request by zinosama**

### CODE REVIEW

**Comment by zinosama** [comment]
@jhanggi

File: app/views/items/index.html.haml [hunk 1]
```diff
@@ -47,10 +47,10 @@
   .row
     .column#menu
       - @categories.each do |category, category_items|
-        %h2.category-title= category.capitalize
+        %h2.category-title= category.capitalize if category.present?
```
Comment (jhanggi) [inline app/views/items/index.html.haml:50]: In what case will `category` be blank?

**Comment by jhanggi** [comment]
@kt32291 you should watch this repo.

**Comment by kt32291** [comment]
Ahh! I thought I was! Thanks @jhanggi

File: app/views/items/index.html.haml (hunk 1 above)
Comment (zinosama) [reply app/views/items/index.html.haml:50]: When you create an item and not specify a category, which is valid as of now. Should I change it?
@jhanggi

**Pull request merged**